icon_validator/workflow/unmarshal.py

```diff
@@ -1,8 +1,7 @@
 import os
 import json
 from icon_validator.workflow.model import Workflow
-from dacite import from_dict
-
+from pydantic import parse_obj_as
```

Comment:
since this was updated to use pydantic, should I change the dataclass reference below?